internal/notification/discord.go

```diff
@@ -49,15 +49,15 @@ const (
 	GRAY       EmbedColors = 10070709 // 99aab5
 )
 
-type DiscordSender struct {
+type discordSender struct {
```

Comment:
I think this still needs to be exported no?

Reply:
it's only used in the notification module so should be fine like this 😄